src/json/any.cr

```diff
@@ -115,6 +115,28 @@ struct JSON::Any
     end
   end
 
+  # Assumes the underlying value is a `Array`
+  # Returns `true` when index given by *index* exists, otherwise `false`
+  def has_key?(index : Int) : Bool
+    case object = @raw
+    when Array
+      object[index]? != nil
```

Comment:
`Array#has_key` doesn't exist,` Any#has_key?` must not exist too – what's the point?